gluten-core/src/main/scala/org/apache/spark/util/memory/TaskMemoryResources.scala

```diff
@@ -30,9 +30,10 @@ import java.util.concurrent.atomic.AtomicLong
 import scala.collection.JavaConverters._
 
 object TaskMemoryResources extends Logging {
+  // And open java assert mode to get memory stack
   val DEBUG: Boolean = {
     SQLConf.get
-      .getConfString("spark.gluten.sql.memory.debug", "false").toBoolean
+      .getConfString("spark.gluten.sql.memory.debug", "true").toBoolean
```

Comment:
true for default?

Reply:
It can be true, still need java VM Options -ea to open Arrow debug mode